src/main/resources/fabric.mod.json

```diff
@@ -8,7 +8,8 @@
     "FX - PR0CESS"
   ],
   "contributors": [
-    "Shnupbups"
+    "Shnupbups",
+    "Space Walker"
```

Comment:
You can put yourself in authors, your already doing tons of work and I doubt your going to stop xD

Reply:
heh fair enough xD